Resources/Prototypes/_Backmen/Flavors/flavors.yml

```
@@ -92,3 +92,18 @@
   description: flavor-complex-bacchus-drink
+
+- type: flavor
+  id: Heavy
+  flavorType: Complex
+  description: flavor-complex-heavy
+
+- type: flavor
+  id: Friendly
+  flavorType: Complex
+  description: flavor-complex-friendly
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)